doc/news/DM-47223.feature.rst

```diff
@@ -0,0 +1,2 @@
+Update ``_wait_hard_point_test_ok`` method in ``MTCS`` to be compatible with
+concurrent executions.
```

Comment:
For documentation in general we adopted something called "semantic line break". Basically it means that each phrase should be in one line. Please, make sure this is all in one line.